### ISSUE TITLE
Improve Lighthouse Reports

### ISSUE THREAD

**Issue by IreTheKID** [mat-1/Repldex#94]
[**Repldex - Lighthouse Report**](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Frepldex.com)

I'm gonna be working on improving the scores for a lot of the failed tests in the Lighthouse Report, specifically in `Accessibility` and `Performance`. This is all frontend stuff so it shouldn't interfere with what y'all have going on with the DB, bot, server, etc.

**Comment by Codemonkey51** [comment]
@mat-1 is there anyway for i.matdoes.dev to have like a `?ttl` parameter so the user can set the ttl

**Comment by jetstream0** [comment]
ttl?

**Comment by Codemonkey51** [comment]
> ttl?

Time to live, iirc it's also known as cache time lmao (so the client caches de image)

**Comment by IreTheKID** [comment]
@Codemonkey51 I was gonna have the server set a `Cache-Control` header in the response. Maybe a week or smth, a year (like Lighthouse suggested) is way too long given how often images foe entries can change.

**Comment by Codemonkey51** [comment]
> @Codemonkey51 I was gonna have the server set a `Cache-Control` header in the response. Maybe a week or smth, a year (like Lighthouse suggested) is way too long given how often images foe entries can change.

we use https://i.matdoes.dev and I am pretty sure it creates a new url for each image (so also each update), and also that's the thing with no cache control and it isn't managed by anything in this repo, so that's why I suggested a ttl parameter for the image

**Comment by IreTheKID** [comment]
Isn't the `Cache-Control` header sent by the server? All we'd need to do is add it to the `aiohttp` response right?

**Comment by Codemonkey51** [comment]
> Isn't the `Cache-Control` header sent by the server? All we'd need to do is add it to the `aiohttp` response right?

We don't control i.matdoes.dev tho (@mat-1 does) so mat would have to change that

**Comment by IreTheKID** [comment]
> > Isn't the `Cache-Control` header sent by the server? All we'd need to do is add it to the `aiohttp` response right?
> 
> We don't control i.matdoes.dev tho (@mat-1 does) so mat would have to change that

OK, that makes sense.

**Comment by mat-1** [comment]
> 
> 
> > Isn't the `Cache-Control` header sent by the server? All we'd need to do is add it to the `aiohttp` response right?
> 
> We don't control i.matdoes.dev tho (@mat-1 does) so mat would have to change that

Ok I added a cache control header (only a day because Cloudflare caches it and if it's too high it'll think no one is viewing the image so it'll just delete it)
